projects/isy-angular-widgets/src/lib/wizard/components/wizard/wizard.component.spec.ts

```diff
@@ -86,7 +86,8 @@ describe('Integration Tests: WizardComponent with Mock Parent', () => {
   let spectator: Spectator<TestComponent>;
   const createComponent = createComponentFactory({
     component: TestComponent,
-    imports: [WizardModule, RouterTestingModule, IncompleteDateModule]
+    imports: [WizardModule, IncompleteDateModule],
+    providers: [WidgetsConfigService, provideRouter([])]
```

Comment:
Muss der WidgetsConfigService in providers?

Reply:
WidgetsConfigService  was deleted